src/main/java/lv/ctco/javaschool/goal/control/DtoConventer.java

```diff
@@ -25,4 +27,13 @@ public static UserLoginDto convertUserToUserLoginDto(User user) {
         return dto;
     }
 
+    public static CommentDto convertCommentToCommentDto(Comment comment) {
+        CommentDto dto = new CommentDto();
+        dto.setUsername(comment.getUser().getUsername());
+        dto.setRegisteredDate(comment.getRegisteredDate().format(DateTimeConverter.formatterDateTime));
```

Comment:
`formatterDateTime` it's field? Why without getters? It's a constant? Why doesn't follow code convention?